runtime/compiler/runtime/JITServerAOTDeserializer.cpp

```diff
@@ -640,6 +642,28 @@ JITServerAOTDeserializer::cacheRecord(const WellKnownClassesSerializationRecord
    }
 
 
+bool
+JITServerAOTDeserializer::cacheRecord(const ThunkSerializationRecord *record,
+                                      TR::Compilation *comp, bool &isNew, bool &wasReset)
+   {
+   if (isResetInProgress(wasReset))
+      return false;
+
+   TR_J9SharedCacheVM *fej9 = (TR_J9SharedCacheVM *)(comp->fe());
```

Comment:
Do we need the cast here? `getJ2IThunk()` and `setJ2IThunk()` are virtual functions.

Reply:
The existing client thunk handling uses `TR_J9VM`, so I'll change that to use `comp->fej9vm()`.